libs/gl-plugin/src/awaitables.rs

```diff
@@ -324,28 +320,10 @@ async fn billboard(
             .status
             .ok_or(Error::Channel("Status not found"))?)
     } else {
-        #[allow(deprecated)]
-        Ok(rpc
-            .call_typed(&ListpeersRequest {
-                id: Some(peer_id),
-                level: None,
-            })
-            .await
-            .map_err(|e| Error::Rpc(e))?
-            .peers
-            .into_iter()
-            .nth(0)
-            .ok_or(Error::Channel("Has no peers list"))?
-            .channels
-            .into_iter()
-            .nth(0)
-            .ok_or(Error::Channel("Has no channels list"))?
-            .into_iter()
-            .filter(|c| c.short_channel_id == Some(scid))
-            .nth(0)
-            .ok_or(Error::Channel("No channel with scid"))?
-            .status
-            .ok_or(Error::Channel("No amount found"))?)
+        return Err(Error::Service(format!(
```

Comment:
Ok, this means we will deprecate any pre-v23.05 node in the `prod` system before rollout. That is intended I assume?

Reply:
For reference, that is currently about <1% of nodes, and <0.01% of active nodes.